src/scss/objects/_form-toggle.scss

```diff
@@ -13,18 +13,18 @@
  */
 
 $formToggle-padding                 : spacing() !default;
-$formToggle-border-color            : $grey--lightest;
-$formToggle-border-color-interact   : $grey--lighter;
+$formToggle-border-color            : $grey--lighter;
+$formToggle-border-color-interact   : $grey--light;
 $formToggle-border-color-checked    : $green;
 $formToggle-border-width            : 1px;
 $formToggle-border-radius           : 8px;
 $formToggle-icon-background         : $green;
-$formToggle-icon-hover-background   : $grey--lighter;
+$formToggle-icon-hover-background   : $grey--light;
 $formToggle-background              : $white;
 $formToggle-large-size              : 48px;
 $formToggle-button-color            : $blue;
```

Comment:
`$btn-secondary-textColor`?

Reply:
Same with this one – I'll wait on the PIE changes to update this